### PR TITLE
Kobo: Enable power/sleep button support for Nia

### PR DESCRIPTION
**Pull request by 3to8-decoder**
see my bug-report/issue: https://github.com/koreader/koreader/issues/9590

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9591)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
If it works. ;-)

File: frontend/device/kobo/device.lua [hunk 1]
```diff
@@ -425,6 +425,7 @@ local KoboLuna = Kobo:new{
     hasFrontlight = yes,
     touch_phoenix_protocol = true,
     display_dpi = 212,
+    power_dev = "/dev/input/event2",
```
Comment (NiLuJe) [inline frontend/device/kobo/device.lua:428]: Nope, potentially too brittle ;).

Check if it works with `automagic_sysfs = true,` (like the Libra 2) instead.

And if it doesn't, get me the output of `ls -lash /dev/input/by-path` ;).

**Review by NiLuJe — CHANGES_REQUESTED** [review]
See above comment (where I pressed the wrong button :x).

I also wouldn't mind the output of

`ntx_hwconfig -l -s /dev/mmcblk0 2>/dev/null`

**Comment by NiLuJe** [comment]
Still waiting on the requested data :).

**Comment by NiLuJe** [comment]
Confirmed via other reports that it's a BD71828 PMIC, much like the other newer hardware revisions in that lineup, so I basically just need to output from `ls -lash /dev/input/by-path` to implement this properly ;).

**Comment by themaz** [comment]
@NiLuJe here is the output:

```
[root@kobo ~]# ls -lash /dev/input/by-path
total 0
     0 drwxr-xr-x    2 root     root          80 May 31 08:31 .
     0 drwxr-xr-x    3 root     root         160 May 31 08:31 ..
     0 lrwxrwxrwx    1 root     root           9 May 31 08:31 platform-bd71828-pwrkey-event -> ../event2
     0 lrwxrwxrwx    1 root     root           9 May 31 08:31 platform-ntx_event0-event -> ../event0
[root@kobo ~]#
```

Also, the `automagic_sysfs = true` worked for me.

**Comment by NiLuJe** [comment]
Superseded by #10520 (thanks for reviving this, @themaz !)

**Pull request closed without merging**